sq/src/sq-usage.rs

```diff
@@ -406,8 +406,8 @@
 //!     extract-cert             Converts a key to a cert
 //!     extract-dsm-secret       Extracts a secret key from Fortanix DSM
 //!     dsm-import
-//!             Imports a Transferable Secret Key into Fortanix DSM
-//!
+//!             Imports a Transferable Public Key (TPK)/Transferable Secret Key
```

Comment:
Nit: The slash is confusing, perhaps simply `Imports a Transferable Secret Key (TSK) or a Transferable Public Key (TPK) into Fortanix DSM